packages/botonic-react/src/message-utils.js

```diff
@@ -0,0 +1,53 @@
+import { INPUT } from '@botonic/core'
+import { MIME_WHITELIST } from './constants'
+
+export const isType = (msgType, type) => msgType === type
+export const isText = msg => isType(msg.type, INPUT.TEXT)
+export const isPostback = msg => isPostback(msg.type, INPUT.POSTBACK)
+export const isAudio = msg => isType(msg.type, INPUT.AUDIO)
+export const isImage = msg => isType(msg.type, INPUT.IMAGE)
+export const isVideo = msg => isType(msg.type, INPUT.VIDEO)
+export const isDocument = msg => isType(msg.type, INPUT.DOCUMENT)
+export const isLocation = msg => isType(msg.type, INPUT.LOCATION)
+export const isContact = msg => isType(msg.type, INPUT.CONTACT)
+export const isCarousel = msg => isType(msg.type, INPUT.CAROUSEL)
+export const isCustom = msg => isType(msg.type, INPUT.CUSTOM)
+export const isButtonMessage = msg => isType(msg.type, INPUT.BUTTON_MESSAGE)
+
+export const MEDIA_TYPES = [
+  INPUT.AUDIO,
+  INPUT.IMAGE,
+  INPUT.VIDEO,
+  INPUT.DOCUMENT,
+]
+
+export const isMedia = message =>
+  MEDIA_TYPES.some(type => isType(message.type, type))
+
+// TODO: Centralize handling attachments in class
+
+export const toBase64 = file =>
+  new Promise((resolve, reject) => {
+    const reader = new FileReader()
+    reader.readAsDataURL(file)
+    reader.onload = () => resolve(reader.result)
+    reader.onerror = error => reject(error)
+  })
+
+export const isAllowedSize = fileSize => {
+  const MAX_MB_SIZE_ALLOWED = 10
+  const maxAllowedBytesSize = MAX_MB_SIZE_ALLOWED * 1024 * 1024
+  if (fileSize > maxAllowedBytesSize) return false
+  return true
+}
+
+export const getAttachmentType = fileType => {
+  return Object.entries(MIME_WHITELIST)
+    .filter(([_, formatsForType]) => formatsForType.includes(fileType))
+    .map(([type, _]) => type)[0]
+}
+
+export const getAcceptedFormats = () =>
```

Comment:
imho this method is very specific for a particular usage. it does not deserve to be in utils. 2 options:
a) convert it to return a single list of all the mimes so that the user only needs to do getFullWhiteList().join(",)
b) move it as private method to where it's used.